### ISSUE TITLE
Problem installing the Visualizer 

### ISSUE THREAD

**Issue by Joul1** [potassco/asprilo#46]
I have an issue using the asprilo visualizer. 
I use the asprilo visualizer via anaconda on windows 10.
Opening, using and saving instances wasn’t a problem but when I tried to save an instance I would always get this Error:

„attributeerror: 'control' object has no attribute 'builder'“.

After a bit of research I thought this might have something to do with the current python version that anaconda uses. But when I tried to update python anaconda would say: „all requested packages already installed“.

I also tried uninstalling and reinstalling the asprilo visualiser itself but when I tried to install it again, it would give the following error:

UnsatisfiableError: The following specifications were found
to be incompatible with the existing python installation in your environment:

Specifications:

  - asprilo-visualizer
-> python[version='>=3.9']

Your python: python=3.7

I also tried to install a specific version of python via “conda install python=3.9”. But doing so, seems to make it stuck in a loop when this line appears:
[04:22<00:03,  2.59it/s]/Examining conflict for conda-content-trust cycler ipython_genutils idna prompt-toolkit backports.shutil_get_terminal_\

I don’t know a lot about how anaconda works so any help is verry appreciated.

**Comment by J-Behrens** [comment]
Hi Julian,
Could be a problem with windows. I recommend to use a Linux distribution like [Ubuntu](https://ubuntu.com/download/desktop). Regarding conda: [Conda Cheat Sheet](https://docs.conda.io/projects/conda/en/4.6.0/_downloads/52a95608c49671267e40c689e0bc00ca/conda-cheatsheet.pdf)

**Comment by Joul1** [comment]
> Hi Julian, Could be a problem with windows. I recommend to use a Linux distribution like [Ubuntu](https://ubuntu.com/download/desktop). Regarding conda: [Conda Cheat Sheet](https://docs.conda.io/projects/conda/en/4.6.0/_downloads/52a95608c49671267e40c689e0bc00ca/conda-cheatsheet.pdf)

Thank you for the fast answer. 
Strangely enough I just managed to update python and reinstall the visualizer. But I have no Idea why it worked all of the sudden.

I still have the issue, that I can save but not load instances with the visualizer, but the Error message is verry different:


TypeError: arguments did not match any overloaded call:
  QRect(): too many arguments
  QRect(int, int, int, int): argument 1 has unexpected type 'float'
  QRect(QPoint, QPoint): argument 1 has unexpected type 'float'
  QRect(QPoint, QSize): argument 1 has unexpected type 'float'
  QRect(QRect): argument 1 has unexpected type 'float'

Do you think this also has something to do with windows? Or is this something I could fix?

**Comment by J-Behrens** [comment]
Hi Julian,
Yes Windows caused problems with the visualizer before. Have you tried to load the instance manually  (not via the terminal)?

**Comment by pobermei** [comment]
>TypeError: arguments did not match any overloaded call:
>QRect(): too many arguments
>QRect(int, int, int, int): argument 1 has unexpected type 'float'
>QRect(QPoint, QPoint): argument 1 has unexpected type 'float'
>QRect(QPoint, QSize): argument 1 has unexpected type 'float'
>QRect(QRect): argument 1 has unexpected type 'float'

@Joul1 Seems like a conversion problem. Are you using python 3.10.x? If so, please try to downgrade to 3.9.x and see if this helps.

**Comment by Joul1** [comment]
Yes, downgrading to 3.9 worked. Thank you so much for your help!

**Comment by pobermei** [comment]
@Joul1: Thanks for your trying this out! 

We have to investigate how we can resolve this incompatibility of pyqt5 and python 3.10 in our code.
